React/game/src/services/loadServices.js

```diff
@@ -0,0 +1,11 @@
+import authService from '@services/authService';
+
+const loadService = async (email, password) => {
+  const response = await authService.login(email, password);
+  const token = response.data.id;
+  const userId = response.data.userId;
+  localStorage.setItem('tokenAuth', token);
+  localStorage.setItem('userId', userId);
+};
```

Comment:
I see that you used 2 services `loadService` and `authService`, one for both logging in and setting the local storage and the other one for the api call...  it's better to have the logging in and the local storage separated. Try creating a `localStorageService` that handles the local storage and the `authService` as you have it right now, then in the onLogin reducer, call these services separately

Reply:
👍